plugins/modules/validate.py

```diff
@@ -61,6 +61,12 @@
     engine: ansible.utils.jsonschema
   vars:
     ansible_jsonschema_draft: draft7
+
+- name: validate configuration with config plugin (see config plugin for criteria examples)
```

Comment:
Do we also want to add a few examples rules in the docs?

Reply:
I'm not sure this file is the place to do that? This example has as much detail as the jsonschema one above, and there are example rules in the sub-plugin, we just need to find a good way to expose them.